include/curl/curl.h

```diff
@@ -1894,6 +1894,9 @@ typedef enum {
   /* set this to 1L to allow HTTP/0.9 responses or 0L to disallow */
   CINIT(HTTP09_ALLOWED, LONG, 285),
 
+  /* TLS session tickets*/
+  CINIT(TLS_USE_SESSION_TICKETS, LONG, 286),
```

Comment:
I think perhaps this should rather be a new bit in the `CURLOPT_SSL_OPTIONS` option. What do you think?

It also needs to be documented. And added to `docs/libcurl/symbols-in-versions`.